Assets/MixedRealityToolkit.Tests/PlayModeTests/InteractableTests.cs

```diff
@@ -37,10 +55,422 @@ public IEnumerator TestAddInteractableAtRuntime()
+            GameObject childObject = GameObject.CreatePrimitive(PrimitiveType.Cylinder);
+            childObject.GetComponent<Renderer>().material.color = Color.blue;
+            childObject.transform.parent = interactableObject.transform;
+            childObject.transform.localScale = new Vector3(0.9f, 1f, 0.9f);
+            childObject.transform.localPosition = new Vector3(0f, 1.5f, 0f);
+            childObject.transform.localRotation = Quaternion.identity;
+            // Only use a collider on the main object
+            GameObject.Destroy(childObject.GetComponent<Collider>());
+
+            translateTargetObject = childObject.transform;
+
+            // Add an interactable
+            interactable = interactableObject.AddComponent<Interactable>();
+
+#if UNITY_EDITOR
+            // Find our states and themes via the asset database
+            Theme cylinderTheme = ScriptableObjectExtensions.GetAllInstances<Theme>().FirstOrDefault(profile => profile.name.Equals($"CylinderTheme"));
+            States defaultStates = ScriptableObjectExtensions.GetAllInstances<States>().FirstOrDefault(profile => profile.name.Equals($"DefaultInteractableStates"));
+
+            interactable.States = defaultStates;
+            InteractableProfileItem profileItem = new InteractableProfileItem();
+            profileItem.Themes = new System.Collections.Generic.List<Theme>() { cylinderTheme };
+            profileItem.HadDefaultTheme = true;
+            profileItem.Target = translateTargetObject.gameObject;
+
+            interactable.Profiles = new System.Collections.Generic.List<InteractableProfileItem>() { profileItem };
+            interactable.ForceUpdateThemes();
+#endif
+
+            // Set the interactable to respond to the requested input action
+            MixedRealityInputAction selectAction = MixedRealityToolkit.InputSystem.InputSystemProfile.InputActionsProfile.InputActions.Where(m => m.Description == selectActionDescription).FirstOrDefault();
+            Assert.NotNull(selectAction.Description, "Couldn't find " + selectActionDescription + " input action in input system profile.");
+            interactable.InputAction = selectAction;
+        }
+
+        /// <summary>
+        /// Instantiates the default interactable buttom.
+        /// </summary>
+        /// <param name="position"></param>
+        /// <param name="rotation"></param>
+        /// <param name="interactableObject"></param>
+        /// <param name="interactable"></param>
+        /// <param name="translateTargetObject"></param>
+        private void InstantiateDefaultInteractablePrefab(Vector3 position, Vector3 rotation, out GameObject interactableObject, out Interactable interactable, out Transform translateTargetObject)
+        {
+            // Load interactable prefab
+            Object interactablePrefab = AssetDatabase.LoadAssetAtPath("Assets/MixedRealityToolkit.Examples/Demos/UX/Interactables/Prefabs/Model_PushButton.prefab", typeof(Object));
```

Comment:
suggest moving that string into a variable defined above so people don't need to hunt for it. I feel like I've done this myself and regretted already not making it a variable (in pressablebuttontests).